sulong/projects/com.oracle.truffle.llvm.parser/src/com/oracle/truffle/llvm/parser/model/functions/FunctionDefinition.java

```diff
@@ -91,6 +93,21 @@ public String getSourceName() {
         return SourceFunction.DEFAULT_SOURCE_NAME.equals(scopeName) ? null : scopeName;
     }
 
+    public String getDisplayName() {
+        // for LLVM from C++, function.name stores the linkage name, but not 'original' C++ name
```

Comment:
For readability, I'd change `for LLVM from C++, ...` to `for LLVM code produced from C++ sources`